app/controllers/admin/schedules_controller.rb

```diff
@@ -82,9 +123,19 @@ def permissions
       @permissions = OpenFoodNetwork::Permissions.new(spree_current_user)
     end
 
-    def sync_subscriptions
+    def sync_subscriptions_update
       return unless params[:schedule][:order_cycle_ids]
 
+      sync_subscriptions
+    end
+
+    def sync_subscriptions_create
```

Comment:
I think this looks better with the for sync_subscriptions_for_create
same for update